website/js/DataTable.js

```diff
@@ -23,10 +23,11 @@ var merge = (...args) => _.extend({}, ...args);
 
 var Lollipop = require('./d3Lollipop');
 
-function setPages({data, count, synonyms}, pageLength) {
+function setPages({data, count, deleted_count, synonyms}, pageLength) { //eslint-disable-line camelcase
```

Comment:
same case as before with eslint -- should decide whether to follow the rule or remove it.